src/plotman/archive.py

```diff
@@ -77,8 +77,13 @@ def compute_priority(phase, gb_free, n_plots):
 
 def get_archdir_freebytes(arch_cfg):
     archdir_freebytes = {}
-    df_cmd = ('ssh %s@%s df -aBK | grep " %s/"' %
-        (arch_cfg.rsyncd_user, arch_cfg.rsyncd_host, arch_cfg.rsyncd_path) )
+    if arch_cfg.mode == 'remote':
+        df_cmd = ('ssh %s@%s df -aBK | grep " %s/"' %
+            (arch_cfg.rsyncd_user, arch_cfg.rsyncd_host, arch_cfg.rsyncd_path) )
+    elif arch_cfg.mode == 'local':
+        df_cmd = ('df -BK | grep " %s/"' % arch_cfg.rsyncd_path )
```

Comment:
I don't know if we want `-a` or not, but do we want local to be different than remote?

```suggestion
	just_df_cmd = 'df -aBK'
    if arch_cfg.mode == 'remote':
        df_cmd = ('ssh %s@%s %s | grep " %s/"' %
            (arch_cfg.rsyncd_user, arch_cfg.rsyncd_host, just_df_cmd, arch_cfg.rsyncd_path) )
    elif arch_cfg.mode == 'local':
        df_cmd = '%s | grep " %s/"' % (just_df_cmd, arch_cfg.rsyncd_path)
```